src/components/Login/LoginForm.js

```diff
@@ -9,25 +9,21 @@ import LoginInput from './LoginInput';
 import 'firebase/auth';
 import '../../styles/Login.scss';
 
-export default class LoginModal extends React.Component {
-  constructor(props) {
-    super(props);
+export default function LoginModal(props) {
+  const { themeColor, textColor } = props;
```

Comment:
Since these props are only used in one function, you can move the destructuring statement to within those functions. Since this component doesn't have a lot of props, this is optional, but can be helpful for clarity in larger components